### ISSUE TITLE
Accessibility issue with missing focus states - header search submit button and page back to top button

### ISSUE THREAD

**Issue by brainiumlabs** [ntgovernment/ntg-web-design-system#1]
Search in header - submit button does not have a visible focus or focus-visible state.

Edit: Page back to top button  -  does it does have focus-visible which can be seen when using keyboard.
